### PR TITLE
LPS-195691 - Automation | Objects | Master

### PR DESCRIPTION
**Pull request by thaiscabrals**
**Ticket:** [LPS-195691](https://liferay.atlassian.net/browse/LPS-195691)
**Test:** ObjectModelBuilder#CanCreateObjectEntry

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by thaiscabrals** [comment]
This PR was previously reviewed [here](https://github.com/liferay-bpm-qa/liferay-portal/pull/1188).

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectModelBuilder.macro [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+	macro AddFolderViaUI(folderLabel = null, labelName = null) {
+		Click(locator1 = "ObjectAdmin#ADD_FOLDER");
+
+		Type(
+			locator1 = "ObjectField#LABEL_INPUT",
+			value1 = ${folderLabel});
+
+		Type(
+			locator1 = "ObjectField#NAME_FIELD",
+			value1 = ${labelName});
```
Comment (patriciajeanperez) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectModelBuilder.macro:13]: @thaiscabrals Let's do folderName so it's the same with the UI

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]
Thank you, @thaiscabrals! I have a suggestion for the macro. Please make the necessary changes and rebase.

**Pull request closed without merging**